src/manageiq_client/api.py

```diff
@@ -322,7 +342,7 @@ class Entity(object):
     SUBCOLLECTIONS = dict(
         service_catalogs={"service_templates"},
         roles={"features"},
-        providers={"tags"},
+        providers={"tags", "custom_attributes"},
```

Comment:
It adds the new "custom_attributes" subcollection in "providers" collection (e.g. "/api/providers/:provider_id/custom_attributes") so it's possible to do e.g. `provider.custom_attributes.action.add({"name": "ca_name1", "value": "ca_value1"}, {"name": "ca_name2", "value": "ca_value2"})`